CHANGELOG.md

```diff
@@ -3,7 +3,43 @@
 
-## Unreleased
+## 0.65.0
+
+This release contains a mix of features and bugfixes.
+
+### Runtime, Tooling, SDK
+
+- ast: Include annotations in rule AST ([#6771](https://github.com/open-policy-agent/opa/pull/6771)) authored by @ashutosh-narkar
+- aws: Always read HTTP response body ([#6734](https://github.com/open-policy-agent/opa/pull/6734)) authored by @johanneslarsson
+- plugins/discovery: Update comparison logic for overrides ([#6723](https://github.com/open-policy-agent/opa/pull/6723)) authored by @ashutosh-narkar
+- plugins/logs: Include http request context in decision logs ([#6693](https://github.com/open-policy-agent/opa/issues/6693)) authored by @ashutosh-narkar reported by @stiidk
+- plugins/rest: Disable the Authorization header for ECR redirects ([6728](https://github.com/open-policy-agent/opa/pull/6728)) authored by @gdlg reported by @vazquezf2000
+- runtime: Fix OpenTelemetry graceful shutdown ([#6651](https://github.com/open-policy-agent/opa/issues/6651)) authored by @nicolaschotard and @David-Wobrock reported by @nicolaschotard
+
+### Topdown and Rego
+
+- topdown: Asserting the `every` domain is a collection type before evaluation ([#6762](https://github.com/open-policy-agent/opa/issues/6762)) authored by @johanfylling reported by @anderseknert
+
+### Miscellaneous
+
+- docs: Add arrays to composite values section ([#6727](https://github.com/open-policy-agent/opa/issues/6727)) authored by @anderseknert reported by @SpecLad
+- docs: Add remainder operator to grammar ([#6767](https://github.com/open-policy-agent/opa/pull/6767)) authored by @anderseknert
+- docs: Fix dynamic metadata object in docs ([#6709](https://github.com/open-policy-agent/opa/pull/6709)) authored by @antonioberben
+- docs: Use best practice package name in test examples ([#6731](https://github.com/open-policy-agent/opa/pull/6731)) authored by @asleire
+- docs: Update query API doc with details about overriding the def decision path ([#6745](https://github.com/open-policy-agent/opa/pull/6745)) authored by @ashutosh-narkar
+- ci: pin GitHub Actions macos runner version and build for darwin/amd64 ([#6720](https://github.com/open-policy-agent/opa/issues/6720)) authored by @suzuki-shunsuke reported by @suzuki-shunsuke
+- Dependency updates; notably:
+  - build(go): bump golang from 1.22.2 to 1.22.3
+  - build(deps): bump aquasecurity/trivy-action from 0.19.0 to 0.21.0
+  - build(deps): bump github.com/containerd/containerd from 1.7.15 to 1.7.17
+  - build(deps): bump github.com/prometheus/client_golang
+  - build(deps): bump golang.org/x/net from 0.24.0 to 0.25.0
+  - build(deps): bump google.golang.org/grpc from 1.63.2 to 1.64.0
+
+### Breaking changes
+
+A new [IsSetStmt](https://www.openpolicyagent.org/docs/latest/ir/#issetstmt) statement has been added to the intermediate representation (IR). 
+This is a breaking change for custom IR evaluators, which must interpret this statement in IR plans generated by this OPA version and later.
```

Comment:
Can we add that nothing changes for Wasm, as long as your Was modules are built with this OPA version.